.env.test

```diff
@@ -5,4 +5,9 @@ FLASK_SKIP_DOTENV=True
 SECRET_KEY=secret-key
 MONGO_DB_USER_NAME=test-username
 MONGO_DB_PASSWORD=test-password
-MONGO_DB_DATABASE_NAME=test-table-name
\ No newline at end of file
+MONGO_DB_DATABASE_NAME=test-table-name
+
+LOGIN_DISABLED=TRUE
+CLIENT_ID=test-client-id
+CLIENT_SECRET=test-client-secret
+OAUTHLIB_INSECURE_TRANSPORT=test
```

Comment:
It would be nice to add these (and any existing but missing) keys to the .env.template as well, to make it more obvious which variables to set for future developers